src/OrchardCore.Modules/OrchardCore.Lists/AdminNodes/ListsAdminNodeNavigationBuilder.cs

```diff
@@ -25,7 +25,7 @@ public class ListsAdminNodeNavigationBuilder : IAdminNodeNavigationBuilder
         private ListsAdminNode _node;
         private ContentTypeDefinition _contentType;
 
-        private const int MaxItemsInNode = 100; // security check
+        private const int _maxItemsInNode = 100; // security check.
```

Comment:
Yes I agree, but Visual Studio shows a warning, because of the new rule we have on private fields (leading underscore required) in the editor config, maybe there is a way to exclude `private const` from this rule. Note: We also have warnings on private localizer fields as `S` and `H`.

Reply:
Okay I reverted the renaming of this const

I suggested here #13953 to ignore the underscore rule for private const.